contrib/linuxarm64v8.Dockerfile

```diff
@@ -0,0 +1,114 @@
+# This dockerfile is meant to cross compile with a x64 machine for a arm64v8 host
+# It is using multi stage build: 
+# * downloader: Download litecoin/bitcoin and qemu binaries needed for c-lightning
+# * builder: Cross compile c-lightning dependencies, then c-lightning itself with static linking
+# * final: Copy the binaries required at runtime
+# The resulting image uploaded to dockerhub will only contain what is needed for runtime.
+# From the root of the repository, run "docker build -t yourimage:yourtag -f contrib/linuxarm64v8.Dockerfile ."
+FROM debian:stretch-slim as downloader
+
+RUN set -ex \
+	&& apt-get update \
+	&& apt-get install -qq --no-install-recommends ca-certificates dirmngr wget \
+     qemu qemu-user-static qemu-user binfmt-support
```

Comment:
You need to install `python3-mako` too

Reply:
why? I am able to build without it.

Reply:
I was not and it's a new dependency which has been added for `generate_wire`. I may be wrong, I'm retesting.

Reply:
At least I was wrong on the place I commented.. It's below at the next `apt install`.

Reply:
Re-tested: I could not build without including `python3-mako` L52.